org/allPRs.ts

```diff
@@ -224,6 +224,8 @@ export const deploySummary = async () => {
 export const rfc327 = () => {
   const semanticFormat = /^(fix|feat|build|chore|ci|docs|style|refactor|perf|test|revert)(?:\(.+\))?!?:.+$/
 
+  const wipFormat = /^(wip|WIP|do not merge|DO NOT MERGE|\[DO NOT MERGE\])(?:\(.+\))?!?:.+$/i
```

Comment:
**suggestion:** I think you don't need to add multiple case sensitive strings like `wip|WIP|` since you already have the case insensitive at the end `+$/i`. 😃

Reply:
**question:** What about draft PRs eg. [DRAFT]? Are those also considered WIP, or Peril is not complaining about them? 😄

Reply:
oh good point! I think a good idea to add a case for those